models/comments.js

```diff
@@ -18,3 +18,20 @@ exports.fetchCommentsByArticleId = (articleId) => {
     }
   );
 };
+exports.addCommentByArticleId = (comment, articleId) => {
+    if(Object.keys(comment).length !== 2)
+    {
+        return Promise.reject({ status: 400, msg: "Bad Request" });
```

Comment:
yeah this won't scale well if the comments table undergoes migrations. (see previous comment on test)